packages/vite/src/node/__tests__/build.spec.ts

```diff
@@ -94,9 +94,9 @@ describe('resolveLibFilename', () => {
 
   test('module package extensions', () => {
     const formatsToFilenames: FormatsToFileNames = [
-      ['es', 'my-lib.es.js'],
+      ['es', 'my-lib.js'],
       ['umd', 'my-lib.umd.cjs'],
-      ['cjs', 'my-lib.cjs.cjs'],
+      ['cjs', 'my-lib.cjs'],
```

Comment:
Maybe you should just add two new cases instead of updating them so we still test old behavior?

Reply:
Wouldn't I still have to update those tests because the filenames have changed? Also those are the tests that I added in #6827.